.env.example

```
@@ -2,4 +2,6 @@
 NEXT_PUBLIC_THIRDWEB_CLIENT_ID=your_thirdweb_client_id
 
 # Contract Addresses
-NEXT_PUBLIC_CONTRACT_ADDRESS=your_smart_contract_address
\ No newline at end of file
+NEXT_PUBLIC_CONTRACT_ADDRESS=your_smart_contract_address
+
+NEXT_PUBLIC_BASE_URL=website_base_url
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)